python/cudf_polars/cudf_polars/callback.py

```diff
@@ -32,6 +33,37 @@
 __all__: list[str] = ["execute_with_cudf"]
 
 
+def enable_uvm(device: int):
```

Comment:
I could be missing some context, but can you simplify this function given that `rmm_mode` depends only on `managed_memory_is_supported` and its never equal to "cuda"?

Reply:
I agree, we don't need to support all the modes like we do in cudf.pandas. In this case we should do a managed pool by default and fall back to the async allocator if not.

Reply:
Done 👍